apps/full-site-editing/package.json

```diff
@@ -96,6 +96,6 @@
 		"classnames": "^2.2.6",
 		"lodash": "*",
 		"moment": "*",
-		"newspack-blocks": "github:Automattic/newspack-blocks#v1.5.0"
+		"newspack-blocks": "github:Automattic/newspack-blocks#1.6.0"
```

Comment:
This looks a bit off to me. any reason for the syntax change?

Reply:
I'd agree that this should retain the `v` as running `yarn run sync:newspack-blocks --release=1.6.0` doesn't work. I'll update the PR accordingly.

Reply:
There seems to be no difference in release/tag names in Newspack Blocks — https://github.com/Automattic/newspack-blocks/tags

Since the intention to change the version number system we use in FSE wasn't communicated in this PR or any commit message, I'll assume it might not be intentional. `v1.6.0` still works well so I will update the code to use it to keep the consistency.

Reply:
Hm, I wonder if there was a release bot that wasn't finished when I created the diff. With the pre-pended `v` the script couldn't find the plugin files when running `yarn run sync:newspack-blocks --nodemodules`